psalm-baseline.xml

```diff
@@ -0,0 +1,145 @@
+    <InternalClass occurrences="2">
+      <code>ConstantNodeChecker::assertValidDefineFunctionCall($node)</code>
+      <code>BuilderHelpers::normalizeValue($constantValue)</code>
+    </InternalClass>
+    <InternalMethod occurrences="2">
+      <code>ConstantNodeChecker::assertValidDefineFunctionCall($node)</code>
+      <code>BuilderHelpers::normalizeValue($constantValue)</code>
+    </InternalMethod>
+  </file>
+  <file src="src/SourceLocator/SourceStubber/ReflectionSourceStubber.php">
+    <InternalClass occurrences="1">
+      <code>BuilderHelpers::normalizeValue($constantReflection-&gt;getValue())</code>
+    </InternalClass>
+    <InternalMethod occurrences="1">
+      <code>BuilderHelpers::normalizeValue($constantReflection-&gt;getValue())</code>
+    </InternalMethod>
+    <PossiblyInvalidArgument occurrences="1">
+      <code>$extensionConstants</code>
+    </PossiblyInvalidArgument>
+    <PossiblyInvalidArrayAccess occurrences="1">
+      <code>$extensionConstants[$constantName]</code>
+    </PossiblyInvalidArrayAccess>
+    <PossiblyInvalidArrayOffset occurrences="1">
+      <code>$extensionConstants[$constantName]</code>
+    </PossiblyInvalidArrayOffset>
+    <PossiblyNullArrayAccess occurrences="1">
+      <code>$extensionConstants[$constantName]</code>
+    </PossiblyNullArrayAccess>
+    <PossiblyNullIterator occurrences="1">
+      <code>$classReflection-&gt;getTraitAliases()</code>
+    </PossiblyNullIterator>
+  </file>
+  <file src="src/SourceLocator/Type/AutoloadSourceLocator.php">
+    <DocblockTypeContradiction occurrences="1">
+      <code>self::$currentAstLocator</code>
+    </DocblockTypeContradiction>
+    <InternalClass occurrences="1">
+      <code>ConstantNodeChecker::assertValidDefineFunctionCall($node)</code>
+    </InternalClass>
+    <InternalMethod occurrences="1">
+      <code>ConstantNodeChecker::assertValidDefineFunctionCall($node)</code>
+    </InternalMethod>
+    <PossiblyInvalidArgument occurrences="1">
+      <code>get_defined_constants(true)['user']</code>
+    </PossiblyInvalidArgument>
+  </file>
+  <file src="src/SourceLocator/Type/ClosureSourceLocator.php">
+    <PossiblyNullReference occurrences="1">
+      <code>getLine</code>
+    </PossiblyNullReference>
+  </file>
+  <file src="src/SourceLocator/Type/Composer/Factory/MakeLocatorForComposerJsonAndInstalledJson.php">
+    <InvalidScalarArgument occurrences="2"/>
+  </file>
+  <file src="src/SourceLocator/Type/Composer/Factory/MakeLocatorForInstalledJson.php">
+    <InvalidScalarArgument occurrences="2"/>
+  </file>
+  <file src="src/SourceLocator/Type/DirectoriesSourceLocator.php">
+    <DocblockTypeContradiction occurrences="1">
+      <code>is_string($directory)</code>
+    </DocblockTypeContradiction>
+  </file>
+  <file src="src/SourceLocator/Type/FileIteratorSourceLocator.php">
+    <DocblockTypeContradiction occurrences="1">
+      <code>$fileInfo instanceof SplFileInfo</code>
```

Comment:
`@param Iterator|SplFileInfo[] $fileInfoIterator` is wrong here: this is an `@param Iterator<SplFileInfo> $fileInfoIterator` instead